engine/modelc/src/java/com/dynamo/bob/pipeline/ModelImporter.java

```diff
@@ -490,6 +508,15 @@ public static void main(String[] args) throws IOException {
         }
 
         System.out.printf("--------------------------------\n");
+
+        System.out.printf("Num Materials: %d\n", scene.materials.length);
+        for (Material material : scene.materials)
+        {
+            DebugPrintMaterial(material, 0);
+        }
+
+        System.out.printf("--------------------------------\n");
+
```

Comment:
Remove debug prints before merging?

Reply:
It is currently the main purpose of the main() function in that class, to debug that ModelUtil can properly load a file and parse it.